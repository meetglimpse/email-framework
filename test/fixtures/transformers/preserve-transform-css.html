<!doctype html>
<html>
  <head>
    <style tailwindcss preserve>
      div {
        @apply uppercase;
      }

      [data-ogsc] .inexistent {
        color: #ef4444;
      }

      div > u + .body .gmail-android-block {
        display: block !important;
      }

      u + #body a {
        color: inherit;
      }
    </style>
  </head>
  <body>
    <div>test</div>
  </body>
</html>
