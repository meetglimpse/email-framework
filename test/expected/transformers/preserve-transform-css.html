<!doctype html>
<html>
  <head>
    <style data-embed="">.block {
        display: block !important;
} .inline {
        display: inline !important;
} .table {
        display: table !important;
} .contents {
        display: contents !important;
} .truncate {
        overflow: hidden !important;
        text-overflow: ellipsis !important;
        white-space: nowrap !important;
} .uppercase {
        text-transform: uppercase !important;
} .lowercase {
        text-transform: lowercase !important;
} .capitalize {
        text-transform: capitalize !important;
} div {
        text-transform: uppercase;
} [data-ogsc] .inexistent {
        color: #ef4444;
      } div > u + .body .gmail-android-block {
        display: block !important;
      } u + #body a {
        color: inherit;
      }
    </style>
  </head>
  <body>
    <div>test</div>
  </body>
</html>
